mops-torch/include/mops/torch/opsa.hpp

```diff
@@ -7,7 +7,6 @@
 
 namespace mops_torch {
 
-/// TODO
```

Comment:
the documentation for this function is still TBD

Reply:
I've added documentation to both mops-torch OPSA and the C++ OPSA